site/content/en/docs/tasks/run/statefulset.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Run StatefulSet"
+linkTitle: "StatefulSet"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a StatefulSet as a Kueue-managed workload.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running StatefulSets.
+Although Kueue does not yet support managing a StatefulSet as a single Workload,
+it's still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the StatefulSet.
```

Comment:
```suggestion
it's still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the StatefulSet.
```
Not sure about it, because we are using Pod group.